src/NuGet.Core/NuGet.Common/NuGetEnvironment.cs

```diff
@@ -25,6 +25,13 @@ public static string GetFolderPath(NuGetFolderPath folder)
                         "NuGet");
                 case NuGetFolderPath.NuGetHome:
                     var userProfile = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
+#if !DNXCORE50
```

Comment:
On when building for `DNXCORE50`, `Environment.GetFolderPath` calls into the code below, which already calls `GetHome`, so no need to call it again.